### PR TITLE
Implementation of Statistics Feature

### PR DESCRIPTION
**Pull request by ruth-naza**
Why:

The users(taskers) need to know how many tasks a taskee is working on and how many taks he has completed

This change addresses the need by:

Creating a profile view page

Using the status column on the TaskManagement table

Creating a statistics_count method that counts the number of Scheduled and Completed Tasks

Implementing a statistic feature for taskers and taskees.

### CODE REVIEW

**Comment by ruth-naza** [comment]
Due to remote and localhost issues with the original ft-statistics-tasks-107901844, I had to create a new branch and push

File: db/schema.rb [hunk 1]
```diff
@@ -59,18 +59,19 @@
...
     t.string   "confirm_token"
-    t.boolean  "confirmed",       default: false
+    t.boolean  "confirmed",            default: false
     t.string   "state"
     t.string   "city"
     t.string   "street_address"
     t.string   "image_url"
-    t.boolean  "has_taken_quiz",  default: false
+    t.boolean  "has_taken_quiz",       default: false
+    t.boolean  "enable_notifications", default: false
```
Comment (atemitope) [inline db/schema.rb:74]: The enable_notifications column should be defaulted to true, that way notifications are only sent to the user unless he decides to disable them

Comment (ruth-naza) [inline db/schema.rb:74]: I didn't work on that in this feature, All did was run rake db:migrate

Comment (ruth-naza) [inline db/schema.rb:74]: Pull from master and run rake db:migrate then check the schema

**Comment by codesword** [comment]
This PR was closed since there was a merge conflict with destination branch.

**Pull request closed without merging**